app/view_models/sign_up_completions_show.rb

```diff
@@ -23,7 +23,7 @@ def initialize(loa3_requested:, decorated_session:, current_user:)
   # rubocop:disable Rails/OutputSafety
   def heading
     if requested_loa == 'loa3'
-      content_tag(:strong, I18n.t('titles.sign_up.verified'))
+      content_tag(:strong, I18n.t('titles.sign_up.verified', app: APP_NAME))
```

Comment:
If someone went an IDP route, is APP_NAME still set to something?

Reply:
yeah, APP_NAME is always set to login.gov in our case.